meeshkan/build/builder.py

```diff
@@ -128,16 +159,56 @@ def update_response(response: Response, mode: UpdateMode, exchange: HttpExchange
         # Need to update existing media type
         existing_media_type = response_content[media_type_key]
         media_type = update_media_type(
-            exchange=exchange, mode=mode, type_key=media_type_key, media_type=existing_media_type)
+            body=exchange.response.body, mode=mode, type_key=media_type_key, media_type=existing_media_type)
     else:
         media_type = build_media_type(
-            exchange=exchange, mode=mode, type_key=media_type_key)
+            body=exchange.response.body, mode=mode, type_key=media_type_key)
 
     new_content: Mapping[str, MediaType] = {media_type_key: media_type}
-    response_content = {**response_content, **new_content }
+    response_content = {**response_content, **new_content}
     return replace(response, headers=new_headers, content=response_content)
 
 
+def update_request_body(request_body: RequestBody, mode: UpdateMode, exchange: HttpExchange) -> RequestBody:
+    """Update request body object.
+
+    RequestBody reference: https://swagger.io/specification/#requestBodyObject
+
+    Arguments:
+        request_body {RequestBody} -- Existing request object.
+        exchange {HttpExchange} -- Request-response pair.
```

Comment:
Missing documentation for `mode`. Documenting all of these is not strictly required as I think it's quite clear from the function signature what it does.